third_party/libusb/webport/src/libusb-to-chrome-usb-adaptor.js

```diff
@@ -106,6 +106,16 @@ GSC.LibusbToChromeUsbAdaptor = class extends GSC.LibusbToJsApiAdaptor {
         chrome.usb.claimInterface, chromeUsbConnectionHandle, interfaceNumber);
   }
 
+  /** @override */
+  async releaseInterface(deviceId, deviceHandle, interfaceNumber) {
+    const chromeUsbDevice = this.getDeviceByIdOrThrow_(deviceId);
+    const chromeUsbConnectionHandle =
+        getChromeUsbConnectionHandle(chromeUsbDevice, deviceHandle);
```

Comment:
nit: It seems these two lines are used together quite often. Does it make sense to extract them into a separate function?